katcp/resource_client.py

```diff
@@ -1114,16 +1124,37 @@ def wait(self, sensor_name, condition_or_value, timeout=5):
             sensor is not present
 
         """
+        if quorum is None:
+            quorum = len(self.clients)
+        elif quorum < 1:
+            quorum = int(math.ceil(quorum * len(self.clients)))
```

Comment:
What happens if `quorum` is exactly 1?  We're not sure if the user intended to wait for 100% of the clients, or just for 1 of them.  As is, the code will assume they meant to wait for 1 client.  Maybe this just needs to be clearer in the docstring.  If they want to wait for 100%, they must use `None`.